ui/client/components/graph/node-modal/editors/expression/Editor.ts

```diff
@@ -73,9 +73,9 @@ const configureValidators = (paramConfig: $TodoType): Array<Validator> => {
   }
 
   return (paramConfig.validators || [])
-    .map(v => validators[v.type])
-    .filter(v => v != null)
-    .map(v => v(paramConfig.editor.possibleValues))
+    .map(v => ({fun: validators[v.type], args: omit(v, ["type"])}))
+    .filter(v => v.fun != null)
+    .map(v => v.fun(v.args))
```

Comment:
this function doesn't look really readable? At very least we should write comment what is going on here, so that adding other validators would be easier?

Reply:
Comment added